api-specificatie/zrc/openapi.yaml

```diff
@@ -1,83 +1,504 @@
               schema:
-                $ref: '#/components/schemas/Fout'
-    parameters: []
-  /zaakinformatieobjecten:
+                $ref: '#/components/schemas/Rol'
+      tags:
+        - rollen
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this Rol.
+        required: true
+        schema:
+          type: integer
+  /statussen:
     post:
-      summary: Voeg een informatieobject toe aan de zaak
-      operationId: zaakinformatieobject_create
-      description: Voeg een informatieobject toe aan een zaak
+      operationId: status_create
+      description: ''
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status'
+      tags:
+        - statussen
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/ZaakInformatieObject'
+              $ref: '#/components/schemas/Status'
+        required: true
+    parameters: []
+  '/statussen/{id}':
+    get:
+      operationId: status_read
+      description: ''
       responses:
-        201:
-          description: ZaakInformatieObject is succesvol aangemaakt.
+        '200':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schema/ZaakInformatieObject'
-        422:
-          description: Opgegeven zaakinformatieobject kan niet verwerkt worden
+                $ref: '#/components/schemas/Status'
+      tags:
+        - statussen
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this status.
+        required: true
+        schema:
+          type: integer
+  /zaakobjecten:
+    post:
+      operationId: zaakobject_create
+      description: Registreer een ZAAKOBJECT relatie.
+      responses:
+        '201':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Fout'
-        default:
-          description: Onverwachte fout tijdens opvoeren informatieobject
+                $ref: '#/components/schemas/ZaakObject'
+      tags:
+        - zaakobjecten
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ZaakObject'
+        required: true
+    parameters: []
+  '/zaakobjecten/{id}':
+    get:
+      operationId: zaakobject_read
+      description: Geef de details van een ZAAKOBJECT relatie.
+      responses:
+        '200':
+          description: ''
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Fout'
+                $ref: '#/components/schemas/ZaakObject'
+      tags:
+        - zaakobjecten
+    parameters:
+      - name: id
```

Comment:
In het RGBZ2 heeft een zaakinformatieobject geen identificatie. Het is een relatieklasse die bestaat bij de gratie van een relatie tussen object en een zaak.
De ID is deze API is dus puur een technisch ID, geen gegeven uit het RGBZ2. Zou dit iets in een UGM moeten worden? @zomwoof